src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -469,6 +473,13 @@ typedef struct
     double* CoutsMarginauxHorairesDeLaReserveParZone;
 } COUTS_MARGINAUX_ZONES_DE_RESERVE;
 
+struct AdequacyPatchParameters
+{
+    bool AdequacyFirstStep;
+    bool Ntc12;
+    bool Ntc11;
```

Comment:
Please rename these two variables to make them more expressive.

Reply:
I will work on this one.

Reply:
Done. Review at #31.